handbook/documentation/index.md

```diff
@@ -12,7 +12,10 @@ We use the [monthly release blog post](https://about.sourcegraph.com/blog) as a
 
 ## Naming and linking documentation pages
 
-1. There is no global index or nav, so all docs should be linked from their parent index page. Every new document should be cross-linked to its related documentation, and linked from its topic-related index, when it exists.
+1. All docs should be linked to from their parent index page.
+1. Every new document should be:
+    - Cross-linked to its related documentation, and linked from its topic-related index, when it exists.
+    - Linked to from the [docs global nav template](https://github.com/sourcegraph/sourcegraph/blob/master/doc/_resources/templates/document.html#L58).
```

Comment:
It is unclear which parts of this document apply to product documentation, handbook documentation or both. This line item is the former I think. Can we clarify this more?

Reply:
I'm addressing this in https://github.com/sourcegraph/about/pull/600#discussion_r383119316